.github/workflows/deploy.yml

```
@@ -44,15 +44,20 @@ jobs:
           fetch-depth: 0
       - run: git checkout ${{ env.git_ref_name }}
         working-directory: ${{github.workspace }}/${{ matrix.repos }}
-        continue-on-error: true
-      - name: 
-        working-directory: ${{github.workspace }}/${{ matrix.repos }}
+        continue-on-error: true 
+      - working-directory: ${{github.workspace }}/${{ matrix.repos }}
         shell: bash
         run: |
           commit_id=$(git rev-parse HEAD)
           mkdir -pv ansible/group_vars/all
           for i in ./ansible/roles/*; do mkdir -pv $i/vars; done
           for i in ./ansible/roles/*; do echo "APP_IMAGE_TAG: $commit_id" >> $i/vars/main.yml; done
+          mkdir -pv ansible/host_vars/brb_host
+          mkdir -pv ansible/host_vars/default_host
+          mkdir -pv ansible/host_vars/nbc_host
+          ansible_varname=$(echo ${{ matrix.repos }} | tr [a-z] [A-Z] | tr - _ | tr \. _ | tr [:blank:] _ )
+          filename=$(echo ${ansible_varname} | tr [A-Z] [a-z] )
+          for i in ./ansible/host_vars/*; do echo "${ansible_varname}_IMAGE_TAG: $commit_id" >> $i/$filename.yml; done
```

Maybe this is easier to read:
```suggestion
          ansible_varname=$(echo ${{ matrix.repos }} | tr [a-z] [A-Z] | tr - _ | tr \. _ | tr [:blank:] _ )
          filename=$(echo ${ansible_varname} | tr [A-Z] [a-z] )
          tennens=(brb default nbc)
          for i in "${tennens[@]}"; do
                    mkdir -pv "ansible/host_vars/${i}_host"
                    echo "${ansible_varname}_IMAGE_TAG: $commit_id" >> "./ansible/host_vars/${i}_host/${filename}.yml"
          done
```

---

no and don't make that what i whant.
Especially for what i plan later on this is not so great, for one of the next steps the folder for the host at host_vars maybe go to be dynamic.
Then the docker image tag should be go to a host explizit, that's why the host folder at the host_vars folder where create. The last thing is the merging of files who are extracted form arcives are overwriting. So it's easyser and better to put the Ansible variable with the docker image tag to seperat files.

---

don't understand the objection, because the suggestion does the same thing